### PR TITLE
[FIX] mail: add "To: No recipient" when there is no followers

### PR DESCRIPTION
**Pull request by phenix-factory**
task-3611153

### CODE REVIEW

File: addons/mail/static/src/core/web/chatter.js [hunk 1]
```diff
@@ -234,6 +234,9 @@ export class Chatter extends Component {
      * @returns {string}
      */
     get toRecipientsText() {
+        if (this.state.thread.recipients.size === 0) {
+            return _t("No recipient found.");
```
Comment (phenix-factory) [inline addons/mail/static/src/core/web/chatter.js:238]: Note: this string already exist in the .pot file: https://github.com/odoo-dev/odoo/blob/saas-16.4-mail-chatter-no-recipients-text-did/addons/mail/i18n/mail.pot#L5364

Comment (alexkuhn) [reply addons/mail/static/src/core/web/chatter.js:238]: The string is already there but it's only registered path is in another file. You should add current file in this POT entry

Comment (phenix-factory) [reply addons/mail/static/src/core/web/chatter.js:238]: Done !

[the remark this answers is not in the document]
Comment (phenix-factory) [reply addons/mail/static/src/core/web/chatter.js:238]: I wanted to reuse an existing/close enough string has this is not master

**Comment by phenix-factory** [comment]
@robodoo r+

**Pull request closed without merging**